etc/clang-format-all.sh

```diff
@@ -12,7 +12,8 @@
 set -o errexit
 set -o pipefail
 
-clang-format --version
+CLANG_FORMAT="${CLANG_FORMAT:-clang-format}"
+"$CLANG_FORMAT" --version
```

Comment:
```suggestion
```suggestion
clang_format_binary="${CLANG_FORMAT_BINARY:-clang-format}"
"${clang_format_binary:?}" --version
```
For consistency with `DOXYGEN_BINARY`, `POETRY_PYTHON_BINARY` (C Driver), etc. + recommend using separate variables for input env var (UPPER_CASE) and local variable (snake_case) + use `${clang_format_binary:?}` to ensure not-null, not-empty when used below (e.g. catch typoes, etc.).